neurox/interpretation/linear_probe.py

```diff
@@ -168,21 +171,24 @@ def _train_probe(
             if use_gpu:
                 inputs = inputs.cuda()
                 labels = labels.cuda()
+            inputs = inputs.float()
             inputs = Variable(inputs)
             labels = Variable(labels)
 
             # Forward + Backward + Optimize
             optimizer.zero_grad()
-            outputs = probe(inputs)
-            if task_type == "regression":
-                outputs = outputs.squeeze()
-            weights = list(probe.parameters())[0]
-
-            loss = (
-                criterion(outputs, labels)
-                + lambda_l1 * l1_penalty(weights)
-                + lambda_l2 * l2_penalty(weights)
-            )
+            
+            with torch.cuda.amp.autocast(): # for mixed precision
```

Comment:
Do we need the autocast() now given that we are not doing mixed precision training?

Reply:
No, removed. Sorry, I overlooked that.